src/main/java/net/mcreator/ui/modgui/PotionGUI.java

```diff
@@ -1,269 +1,153 @@
-		potion.renderStatusInHUD = renderStatusInHUD.isSelected();
-		potion.onStarted = onStarted.getSelectedProcedure();
-		potion.onActiveTick = onActiveTick.getSelectedProcedure();
-		potion.onExpired = onExpired.getSelectedProcedure();
-		return potion;
-	}
-
-	@Override public @Nullable URI getContextURL() throws URISyntaxException {
-		return new URI(MCreatorApplication.SERVER_DOMAIN + "/wiki/how-make-potion");
-	}
-
-}
+/*
+ * MCreator (https://mcreator.net/)
+ * Copyright (C) 2012-2020, Pylo
+ * Copyright (C) 2020-2021, Pylo, opensource contributors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package net.mcreator.ui.modgui;
+
+import net.mcreator.element.types.Potion;
+import net.mcreator.ui.MCreator;
+import net.mcreator.ui.component.util.PanelUtils;
+import net.mcreator.ui.help.HelpUtils;
+import net.mcreator.ui.init.L10N;
+import net.mcreator.ui.minecraft.potions.JPotionList;
+import net.mcreator.ui.validation.AggregatedValidationResult;
+import net.mcreator.ui.validation.ValidationGroup;
+import net.mcreator.ui.validation.component.VTextField;
+import net.mcreator.ui.validation.validators.TextFieldValidator;
+import net.mcreator.util.StringUtils;
+import net.mcreator.workspace.elements.ModElement;
+
+import javax.annotation.Nonnull;
+import javax.swing.*;
+import java.awt.*;
+
+public class PotionGUI extends ModElementGUI<Potion> {
+
+	private final VTextField potionName = new VTextField(40);
+	private final VTextField splashName = new VTextField(40);
+	private final VTextField lingeringName = new VTextField(40);
+	private final VTextField arrowName = new VTextField(40);
+	private final JPotionList effectList = new JPotionList(mcreator);
+
+	private final JPanel effects = new JPanel(new GridLayout(0, 1, 5, 5));
+
+	private final ValidationGroup page1group = new ValidationGroup();
+
+	public PotionGUI(MCreator mcreator, @Nonnull ModElement modElement, boolean editingMode) {
+		super(mcreator, modElement, editingMode);
+		this.initGUI();
+		super.finalizeGUI();
+	}
+
+	@Override protected void initGUI() {
+		JPanel pane3 = new JPanel(new BorderLayout());
+		pane3.setOpaque(false);
+
+		JPanel northPanel = new JPanel(new GridLayout(4, 2, 0, 2));
+		northPanel.setOpaque(false);
+
+		northPanel.add(HelpUtils.wrapWithHelpButton(this.withEntry("potion/display_name"),
```

Comment:
display_name is not correct anymore, split this into 4 tips and update the tip text

Reply:
What did you change for it not to be correct anymore?

Reply:
Me, nothing. But it says that MCreator will automatically add "tipped arrow" for example, which is not correct, this text needs to be now entered manually so this needs to be updated.